README.md

```diff
@@ -94,7 +94,51 @@ export LOG_LEVEL=INFO
    `mvn clean compile exec:java@aaa-server`
 4. The server will be up in Port 8443.
 
+### JAR based
+1. Install java 11 and maven
+2. Set Environment variables
+```
+export AUTH_URL=http://<auth-domain-name>
+export LOG_LEVEL=INFO
+```
+3. Use maven to package the application as a JAR
+   `mvn clean package -Dmaven.test.skip=true`
+4. 2 JAR files would be generated in the `target/` directory
+    - `iudx.aaa.server-cluster-0.0.1-SNAPSHOT-fat.jar` - clustered vert.x containing micrometer metrics
+    - `iudx.aaa.server-dev-0.0.1-SNAPSHOT-fat.jar` - non-clustered vert.x and does not contain micrometer metrics
+
+#### Running the clustered JAR
```

Comment:
Maybe we can add a point here. 
1. Need to setup the zookeeper : https://zookeeper.apache.org/doc/r3.3.3/zookeeperStarted.html
2. Edit the config file option "zookeeper" should point to  ip/domain the zookeeper is hosted. The aaa server assumes the default port 2181 for zookeeper related communication and 5701 port for hazelcast communication between different vertx instances.